edm4eic.yaml

```
@@ -244,6 +244,27 @@ datatypes:
   ## ==========================================================================
   ## Calorimetry
   ## ==========================================================================
+
+  edm4eic::RawCALOROCHit:
+    Description: "Raw hit from a CALOROC/HGCROC chip"
+    Author: "D. Anderson, S. Joosten, N. Novitzky"
+    Members:
+      - uint32_t type              // Chip type, 0 - type 1A, 1 - type 1B
+      - uint64_t cellID            // Detector specific (geometrical) cell id
+      - int32_t  samplePhase       // Phase of samples in [# samples], for synchronizing across chips
+      - int32_t  timeStamp         // [TDC counts]
+    VectorMembers:
+      - uint32_t amplitude         // Iff type == 0, amplitudes of waveforms are stored here [ADC counts]
+      - uint32_t lowGainAmplitude  // Iff type == 1, low gain amplitudes of waveforms are stored here [ADC counts]
+      - uint32_t highGainAmplitude // Iff type == 1, high gain amplitudes of waveforms are stored here [ADC counts]
```

```suggestion
      - uint32_t amplitude         // Iff type == 0, amplitudes of waveform are stored here [ADC counts]
      - uint32_t lowGainAmplitude  // Iff type == 1, low gain amplitudes of waveform are stored here [ADC counts]
      - uint32_t highGainAmplitude // Iff type == 1, high gain amplitudes of waveform are stored here [ADC counts]
```
Only one waveform is stored? (not necessarily a single "pulse") If not, how do we distinguish them.

---

Oh! Good catch! That's correct: it should be only 1 waveform...